requirements-all.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
-bitsandbytes==0.41.0    # quantization
+bitsandbytes>=0.41.2   # quantization
```

Comment:
The problem with pinning is that there's no automation (like dependabot) that periodically checks if dependencies can be upgraded.

Also, can you fix the formatting?